src/Nox.Solution/Models/Domain/Entities/Relationships/EntityRelationship.cs

```diff
@@ -38,6 +38,10 @@ public class EntityRelationship : DefinitionBase
     [Description("This boolean controls whether this side of the relationship is exposed in the generated code and ODATA endpoints.")]
     public bool CanNavigate { get; internal set; } = true;
 
+    [Title("Determines whether this side of the relationship should contain contain a foreign key.")]
```

Comment:
this is to be ignored, this is not a user option but a computed property, use YamlIgnore and remove the other attribute